keps/prod-readiness/sig-testing/2420.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2420
 beta:
-  approver: "@johnbelamaric"
\ No newline at end of file
+  approver: "@johnbelamaric"
```

Comment:
i think these PRR approvals need to be held until the kep is actually reviewed by PRR

Reply:
I interpreted as need PRR approval if I'm going to change status or stage, even retroactively. Some of the tests will fail if not checked in concurrently

I wanted to make sure other stuff was ok before calling for PRR

Reply:
i didn't think you wanted all of these held up (since it's in the same PR) for PRR review for something like 2420 which is going stable in 1.23 (as opposed to things like 2290 which is a retroactive kep update), but if you're a-ok with waiting, fine with me :)

Reply:
hm, good point... how about I move to implemented at previous milestone, and use a separate PR for proposing implementable at latest milestone

Reply:
Broke out v1.23 KEP graduations to https://github.com/kubernetes/enhancements/pull/2871